app/models/contact.rb

```diff
@@ -14,6 +14,7 @@ class Contact < ApplicationRecord
   has_paper_trail
 
   validates :first_name, presence: true
+  validates :nhs_number, uniqueness: { case_sensitive: false }
```

Comment:
think this needs `, allow_blank: true` added otherwise it will think its a duplicate when blank